docs/csharp/language-reference/keywords/snippets/InitExample5.cs

```diff
@@ -0,0 +1,13 @@
+﻿class Person_InitExampleFieldProperty
+{
+    public int YearOfBirth
+    {
+        get;
```

Comment:
I assume that these won't be compile issues once .NET 9 is released? Or could we set the `<LangVersion>` in this PR to get these green?

Reply:
The `LangVersion` is set to preview, but the bits aren't in the current preview for the Docker SDK image.